.changeset/sweet-elephants-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"strapi-cms": patch
+---
+
+fix icons from Features component on Next.js plus fix footer socials link on strapi
```

Comment:
Rather than having a description that includes all the changes in different packages, it is better to have a dedicated description per package. So it is best to create a changeset for each package. Just put them in different changeset files